### DISCUSSION TITLE
How old was Emma?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#803] [Galaxy Trivia] [answered]
At a children's birthday party, we noted the ages of the children present.
Oliver was 3, James and Penelope were 4, Jessica and William were 5.

**Comment by peterbaccaro** [comment]
Based on the information given, Emma's age cannot be determined. 

If you tried to solve this mathematically by giving each letter a numerical value one possible outcome is as follows: 

![image](https://github.com/ParanoidUser/yolo/assets/74666079/28b046fe-bcba-4a90-ab1f-5a66c0b8d30f)

This would result in Emma being **3** (1+0+0+2).

**Comment by ParanoidUser** [comment]
**Tiny Hint:** Look at the vowels (AEIOU).

**Reply by peterbaccaro** [reply]
Emma is **4**.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/138df863-c54f-4e70-8297-88f1818c4bac)

**Reply by ParanoidUser** [reply]
Warm! 🫢 Now let's see what role consonants play here. 🧮

**Reply by peterbaccaro** [reply, marked as answer]
Another way to calculate that also works is to calculate the number of vowels and constants and to use this formula:

![image](https://github.com/ParanoidUser/yolo/assets/74666079/3ec136d8-2b0e-43dd-b3b3-9fb9fd87ab35)

This would result in EMMA being **2**.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/fdd4a032-8e1a-4cfd-81da-f63fff51efb7)

**Reply by ParanoidUser** [reply]
Bingo! 👏 Or another way to calculate it `2 * <numConsonants> - <numVowels>`

**Comment by ParanoidUser** [comment]
@JCluzet / @codeleesh, any ideas? 😶‍🌫️
